lib/tasks/mastodon.rake

```diff
@@ -36,7 +36,7 @@ namespace :mastodon do
   desc 'Refetch missing avatar and header'
   task refetch_avatar_header: :environment do
     Account.remote.find_each do |account|
```

Comment:
The batch size for find_each is a default of 1000, it would be a good idea to limit the batch size as it is too high and causes performance problems according to my tests. i.e.
`Account.remote.find_each(batch_size: 8) do ...`